app/views/posts/_form.html.erb

```diff
@@ -4,7 +4,8 @@
       <div class="card rounded mb-3">
       <div class="form-group">
         <%= label_tag :Community %>
-        <%= form.select :community_id, @my_communities.collect {|option| [option.name, option.id]}, prompt: "Select Community", selected: params[:community_id] , class: "form-select" %>
+        <%= form.select :community_id, @my_communities.collect {|option| [option.name, option.id]}, prompt: "Select Community", selected: selected_community(@my_communities.pluck(:id)) , class: "form-select "+is_error(@post,(:community_id)).to_s %>
```

Comment:
can you move a part of this to helper to simplify?

Reply:
already selected_community and is_error are methods in the helper @iMagesh

Reply:
I was thinking if you can move the "form-select" and .to_s part also into the helper method so it will look better and simple

Reply:
Refactored code @iMagesh